src/openforms/conf/base.py

```diff
@@ -191,7 +191,7 @@
     # 'django.middleware.locale.LocaleMiddleware',
     "corsheaders.middleware.CorsMiddleware",
     "django.middleware.common.CommonMiddleware",
-    "django.middleware.csrf.CsrfViewMiddleware",
+    # "django.middleware.csrf.CsrfViewMiddleware",
```

Comment:
this is a **massive** security risk, I'm hoping it's an accident and not intentional

Reply:
Yes, this was me doing a bit of debugging and accidentally not reverted.